applications/GeoMechanicsApplication/tests/test_steady_state_groundwater_flow.py

```diff
@@ -58,12 +58,9 @@ def calculate_outflow_discharge(self, simulation):
         """
 
         hydraulic_discharge = test_helper.get_hydraulic_discharge(simulation)
-        sumValue = 0
-        for value in hydraulic_discharge:
-            if (value > 0):
-                sumValue = sumValue + value
 
-        return sumValue
+        from functools import reduce
+        return reduce(lambda a,b: a+b if b>0 else a, hydraulic_discharge, 0)
```

Comment:
@Vahid-Galavi I am quite sure it works as I tested it with a simpler example but please make sure you also check it once

Reply:
@philbucher It works before pushing. It works. Thanks again for the suggestion.